sycl/test-e2e/Graph/exception_unsupported_backend.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests the ability to finalize a empty command graph
+// without submitting the graph.
+
+#include "graph_common.hpp"
+
+int GetUnsupportedBackend(const sycl::device &Dev) {
+  // Return 1 if the device backend is "cuda" or 0 else.
```

Comment:
>  // Return 1 if the device backend is "cuda" or 0 else.

cuda is quite specific, would say "if the device backend is unsupported" instead to be more generic